Add files via upload

marigo8 opened the pull request:
create introduction file

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

marigo8 committed:
Add files via upload

create introduction file

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+hi! I am Mario!
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

marigo8 committed:
add favorite path

marigo8 committed:
add deadly skills and fix lines

marigo8 committed:
Merge pull request #1 from marigo88/deadly_skills

add deadly skills and fix lines

marigo8 committed:
Merge pull request #2 from marigo88/master

Merge pull request #1 from marigo88/deadly_skills

marigo8 committed:
Merge pull request #3 from marigo88/deadly_skills

Merge pull request #2 from marigo88/master

marigo8 committed:
add Excel to deadly skills

The first submission wouldn't work for mysterious reasons so I am trying it again.

marigo8 committed:
add Killing history using git rebase

marigo8 committed:
Merge pull request #4 from marigo88/deadly_skills

add Excel to deadly skills

codeschool-kiddo committed:
Added new rule about coffee

marigo8 committed:
fixed spelling mistakes

marigo8 committed:
Merge pull request #5 from marigo88/new_rules

Too many people are spilling coffee

marigo8 committed:
replace assassins with bad coding

marigo8 committed:
Merge pull request #6 from marigo88/release_branch_1.0

Release branch 1.0

marigo8 committed:
add redundancy to kill list

marigo8 committed:
remove redundancy from master branch

marigo8 committed:
add forgotten semicolons to list

marigo8 committed:
fixes #7

marigo8 committed:
Merge pull request #8 from marigo88/kill_list

Kill list

marigo8 committed:
All members should read the rules
